packages/ui/components/combobox/test/lion-combobox.test.js

```diff
@@ -3018,6 +3063,37 @@ describe('lion-combobox', () => {
       expect(el.opened).to.equal(true);
     });
 
+    it('resets the inputNode.value and listbox options on [Enter]', async () => {
```

Comment:
```suggestion
    it('clears texbox after selection of a new item', async () => {
```

Reply:
Right now, it does not work on click for instance...

Reply:
And listbox options should not be cleared right? (they are not atm, but previous it statement seemed to suggest they should)

Reply:
https://github.com/user-attachments/assets/2488d593-059e-48af-8dcc-b37a415d422d

Reply:
Also the listbox options should be cleanup, otherwise only a limited amount of options with highlighting are shown.